media-video/motion/files/motion.initd-r4

```diff
@@ -0,0 +1,16 @@
+#!/sbin/openrc-run
+# Copyright 1999-2017 Gentoo Foundation
```

Comment:
`2020` and switch `Foundation` to `Authors`.

Reply:
Ok, will do